apps/sync/index.html

```diff
@@ -11,4 +11,7 @@
        are still in development.
      </p>
   </body>
+  <script src="js/fxsync-webcrypto/stringconversion.js"></script>
+  <script src="js/fxsync-webcrypto/keyderivation.js"></script>
+  <script src="js/fxsync-webcrypto/fxsyncwebcrypto.js"></script>
```

Comment:
We can probably lazy load these scripts. We should only be needing them when there is something to push or when we pulled something from Syncto.